cves/CVE-2013-4310.yml

```diff
@@ -45,7 +45,9 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  Stuts 2 controls actions using a special prefix parameter, which can be
```

Comment:
```suggestion
  Struts 2 controls actions using a special prefix parameter, which can be
```

Reply:
A little more detail here would be nice. How can these control actions be tweaked?